server/container/util/pom.xml

```diff
@@ -75,6 +75,10 @@
             <groupId>io.projectreactor</groupId>
             <artifactId>reactor-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.projectreactor.addons</groupId>
```

Comment:
is it used?

Reply:
Yes, our custom `RetryWithAsyncCallback` is based on addons `AbstractRetry`